docs/Generics/generics.tex

```diff
@@ -893,6 +893,7 @@ \section{Delayed Parsing}
 
 \begin{Verbatim}
 // c.swift
+import Foundation // @objc attribute cannot be used without importing module 'Foundation'
```

Comment:
Can you remove the comment?

Reply:
Actually this is kind of weird, because increasing the length of this listing changes the page layout for the rest of the chapter, introducing some awkward page breaks. I'll see if there is a way to re-organize the material to avoid that later, but for now let's skip adding this import here.